kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -511,6 +516,36 @@ def kube_job_status_succeeded(self, metric, scraper_config):
                     tags.append(self._format_tag(label_name, label_value, scraper_config))
             self.job_succeeded_count[frozenset(tags)] += sample[self.SAMPLE_VALUE]
 
+    def kube_job_status_start_time(self, metric, scraper_config):
+        for sample in metric.samples:
+            tags = [] + scraper_config['custom_tags']
+            for label_name, label_value in iteritems(sample[self.SAMPLE_LABELS]):
+                tags.append(self._format_tag(label_name, label_value, scraper_config))
+            self.job_start_time[frozenset(tags)] += int(sample[self.SAMPLE_VALUE])
+
+    def kube_job_status_completion_time(self, metric, scraper_config):
+        for sample in metric.samples:
+            tags = [] + scraper_config['custom_tags']
+            for label_name, label_value in iteritems(sample[self.SAMPLE_LABELS]):
+                tags.append(self._format_tag(label_name, label_value, scraper_config))
+            self.job_completion_time[frozenset(tags)] += int(sample[self.SAMPLE_VALUE])
+
+    def kube_job_status_execution_time(self, scraper_config):
+        metric_name = scraper_config['namespace'] + '.job.execution_time'
+        for job_tags, completion_time in iteritems(self.job_completion_time):
+            if job_tags in self.job_start_time:
+                start_time = self.job_start_time[job_tags]
+                tags = [] + scraper_config['custom_tags']
+                for job_tag in list(job_tags):
+                    label = job_tag.split(":")
+                    label_name = label[0]
+                    label_value = label[1]
+                    if label_name == 'job' or label_name == 'job_name':
```

Comment:
You can do `if label_name in ('job', 'job_name'):`

Reply:
Fixed it